Project.toml

```diff
@@ -22,6 +23,7 @@ DataStructures = "0.18"
 Dates = "1"
 InfrastructureSystems = "2"
 JSON3 = "1"
+KLU = "0.6.0"
```

Comment:
Can you specify this version more flexibly? See e.g. https://github.com/NREL-Sienna/PowerNetworkMatrices.jl/blob/0239dfcd140bc60415c2e0abdf3105dab13e3af7/Project.toml#L23